### DISCUSSION TITLE
.env file location in sveltekit quickstart?

### DISCUSSION THREAD

**Discussion by mtndan** [supabase/supabase#11359] [Questions]
Hi there,

I'm new to Sveltekit and following the quickstart here https://supabase.com/docs/guides/getting-started/tutorials/with-sveltekit.

In the step about setting up the .env file, where should that go in the filesystem? I tried putting it at the root of my project and am unable to start the app. 

Thank you in advance,
Dan

**Comment by davepar** [comment]
I have the .env file in the root of my project, next to package.json. That should work. I seem to remember that the quickstart was out of date the last time I tried it, and that was before the 1.0 release of SvelteKit. The readme for the auth helper has better info: https://github.com/supabase/auth-helpers/tree/main/packages/sveltekit

**Comment by ioucyf** [comment]
With the file .env in your project's root directory, try the follow:


```
import { createClient } from '@supabase/auth-helpers-sveltekit'

import { PUBLIC_SUPABASE_URL,  PUBLIC_SUPABASE_ANON_KEY} from '$env/static/public'

export const supabase = createClient(PUBLIC_SUPABASE_URL, PUBLIC_SUPABASE_ANON_KEY)
```
